src/schemas/review.ts

```diff
@@ -25,3 +25,7 @@ export const reviewSubmitSchema = createInsertSchema(reviews, {
   updatedAt: true,
   referral: true,
 });
+
+export const referApplicantSchema = createInsertSchema(reviews, {
+  applicantUserId: z.string().min(1).max(255),
+});
```

Comment:
I don't think we need to check the length of the applicantUserId here :)). Another issue here is that this requires all other fields as well in the reviews table. I think we can just do this:

```suggestion
export const referApplicantSchema = createInsertSchema(reviews).pick({ applicantUserId: true });
```

Reply:
appreciate the help 🙏